configuration/api.go

```diff
@@ -513,3 +513,14 @@ func (a *API) UpdateTag(name string, groupIDs []int) error {
 		GroupIDs: groupIDs,
 	}, &emptyResponse{})
 }
+
+// Lists properties of groups
+func (a *API) ListGroupsProperties(namespace string, namespacePrefix string, groupIDs []int) ([]listGroupsPropertiesResponse, error) {
```

Comment:
We shouldn't return private `struct` to the developers. I think you could create `GroupProperties`:

```go
type GroupProperties struct {
	ID         int32
	Properties Properties
}
```

Reply:
Also, more often will be used to property `groupIDs` than `namespace`. I suggest to reorder fields: `groupIDs` -> `namespace` -> `namePrefix` (it's not `namespacePrefix`).

Reply:
Fixed acc. to suggestions.